shell/client/shell.html

```diff
@@ -75,12 +75,13 @@
   {{/if}}
   {{#if currentUser}}
     {{>sandstormTopbarItem name="notifications" topbar=globalTopbar
-                           template="notifications" popupTemplate="notificationsPopup"}}
+                           template="notifications" popupTemplate="notificationsPopup"
+                           priority=-1}}
   {{/if}}
   {{!-- admin alerts --}}
   {{#with adminAlert}}
     {{#if adminAlertIsTooLarge}}
-      {{#sandstormTopbarItem name="admin-alert" topbar=globalTopbar}}
+      {{#sandstormTopbarItem name="admin-alert" topbar=globalTopbar priority=-2}}
```

Comment:
Until the flexbox changes in https://github.com/sandstorm-io/sandstorm/pull/1190 get merged, I don't know that the `priority=` parameter does anything. But OK!